rxandroidble/src/main/java/com/polidea/rxandroidble/ConnectionSetup.java

```diff
@@ -34,6 +34,10 @@ private ConnectionSetup(boolean autoConnect, boolean suppressOperationCheck) {
 
 
         /**
+         * Autoconnect concept may be misleading at first glance. In cases when the BLE device is available and it is advertising constantly
+         * you won't need to use autoconnect. Use autoconnect for connections where the BLE device is not advertising at
+         * the moment of #establishConnection call.
```

Comment:
`{@link com.polidea.rxandroidble.RxBleConnection#establishConnection(ConnectionSetup)}` instead of `#establishConnection`